GCRCatalogs/dc2_metacal.py

```diff
@@ -0,0 +1,102 @@
+"""
+DC2 Metacal Catalog Reader
+"""
+
+import os
+import numpy as np
+from .dc2_dm_catalog import DC2DMCatalog
+
+__all__ = ['DC2MetacalCatalog']
+
+
+class DC2MetacalCatalog(DC2DMCatalog):
+    r"""DC2 Metacal Catalog reader
+
+    Parameters
+    ----------
+    base_dir          (str): Directory of data files being served, required
+    filename_pattern  (str): The optional regex pattern of served data files
+    use_cache        (bool): Cache read data in memory
+    is_dpdd          (bool): File are already in DPDD-format.  No translation.
+
+    Attributes
+    ----------
+    base_dir          (str): The directory of data files being served
+
+    Notes
+    -----
+    """
+    # pylint: disable=too-many-instance-attributes
+
+    FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+    FILE_PATTERN = r'metacal_tract_\d+\.parquet$'
+    SCHEMA_FILENAME = 'schema.yaml'
+    META_PATH = os.path.join(FILE_DIR, 'catalog_configs/_dc2_metacal_meta.yaml')
+
```

Comment:
Remove line.  Just one blank line between methods with a class definition.

Here and below.